torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -22,13 +22,30 @@ template <typename Op>
 void visitBinaryOp(
     const BinaryOpNode<Op>* v,
     const std::string& op_str,
-    IRPrinter* printer) {
+    IRPrinter* printer,
```

Comment:
Also since this changes our invariance, please also check other print outs, such as ternary ops, like IfThenElse.

Reply:
Checked the others.  IfThenElse needed some adjustments, but the rest were fine.